source/extensions/retry/host/omit_host_metadata/omit_host_metadata.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "envoy/upstream/retry.h"
+#include "envoy/upstream/upstream.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Retry {
+namespace Host {
+
+class OmitHostsRetryPredicate : public Upstream::RetryHostPredicate {
+public:
+  OmitHostsRetryPredicate(envoy::config::core::v3alpha::Metadata metadata_match_criteria)
```

Comment:
make the constructor `explicit` and pass the metadata by const ref (or move the value into the member variable)

Reply:
Done.